src/rules/camelcase.rs

```diff
@@ -60,9 +69,16 @@ let firstName = "Ichigo";
 const FIRST_NAME = "Ichigo";
 const __myPrivateVariable = "Hoshimiya";
 const myPrivateVariable_ = "Hoshimiya";
+const { last_name: lastName } = obj;  // where `obj` is defined externally
```

Comment:
Whether `obj` is defined externally or internally doesn't change the linter behavior for this line, so I think this comment could be removed 👍

Reply:
Agreed.